src/panoptes/pocs/focuser/focuser.py

```diff
@@ -343,49 +340,44 @@ def _autofocus(self,
             focus_type = 'coarse'
 
         initial_focus = self.position
-        self.logger.debug("Beginning {} autofocus of {} - initial position: {}",
-                          focus_type, self._camera, initial_focus)
+        self.logger.debug(f"Beginning {focus_type} autofocus of {self._camera} - initial position: {initial_focus}")
 
         # Set up paths for temporary focus files, and plots if requested.
         image_dir = self.get_config('directories.images')
         start_time = current_time(flatten=True)
-        file_path_root = os.path.join(image_dir,
-                                      'focus',
-                                      self._camera.uid,
-                                      start_time)
+        file_path_root = os.path.join(image_dir, 'focus', self._camera.uid, start_time)
 
         self._autofocus_error = None
 
-        dark_thumb = None
+        dark_thumbnail = None
         if take_dark:
-            dark_path = os.path.join(file_path_root,
-                                     '{}.{}'.format('dark', self._camera.file_extension))
-            self.logger.debug('Taking dark frame {} on camera {}'.format(dark_path, self._camera))
+            dark_path = os.path.join(file_path_root, f'dark.{self._camera.file_extension}')
+            self.logger.debug(f'Taking dark frame {dark_path} on camera {self._camera}')
             try:
-                dark_thumb = self._camera.get_thumbnail(seconds,
-                                                        dark_path,
-                                                        thumbnail_size,
-                                                        keep_file=True,
-                                                        dark=True)
+                dark_thumbnail = self._camera.get_thumbnail(seconds,
+                                                            dark_path,
+                                                            thumbnail_size,
+                                                            keep_file=True,
+                                                            dark=True)
                 # Mask 'saturated' with a low threshold to remove hot pixels
-                dark_thumb = mask_saturated(dark_thumb,
-                                            threshold=0.3,
-                                            bit_depth=self.camera.bit_depth)
+                dark_thumbnail = mask_saturated(dark_thumbnail,
```

Comment:
Do we actually need/want to be masking parts of the dark image? Would it be more appropriate to apply this once to the dark-subtracted frame?